packages/client/hmi-client/package.json

```diff
@@ -73,6 +73,7 @@
 		"vue": "3.3.13",
 		"vue-feather": "2.0.0",
 		"vue-gtag": "2.0.1",
+		"vue-markdown-render": "^2.2.1",
```

Comment:
This library looks fairly simple - have we done any evaluation here on what it adds? 
It may be better to just work with "markdown-it" directly instead of having to maintain dozens and dozens of depfendencies.

Reply:
so i am just using `markdown-it` now. seems like the dependencies added to the `yarn.lock` are slightly less and more up to date actually